qiskit/circuit/quantumcircuit.py

```diff
@@ -4341,6 +4344,11 @@ def map_calibration(qubits, parameters, schedule):
         )
         return None if inplace else target
 
+    def has_control_flow_op(self) -> bool:
```

Comment:
@jakelishman's earlier comment is on point here this is past feature freeze so we shouldn't make this public if we want to add it for 2.0. There is a path to fix the regression in python as he suggested, but with my other comments here doing the iteration in rust is faster. We could also still iterate in rust if we did the set intersection with `count_ops()` python side (like is currently in the rust code).